toolkit/tools/scheduler/scheduler.go

```diff
@@ -557,21 +557,44 @@ func buildAllNodes(stopOnFailure, canUseCache bool, packagesToRebuild, testsToRe
 	schedulerutils.PrintBuildSummary(builtGraph, graphMutex, buildState, allowToolchainRebuilds, licenseChecker)
 	schedulerutils.RecordBuildSummary(builtGraph, graphMutex, buildState, *outputCSVFile)
 
+	printErr := schedulerutils.PrintHiddenBuildBlockers(builtGraph, graphMutex, buildState, goalNode)
+	if printErr != nil {
+		logger.Log.Warnf("Failed to print hidden build blockers:\n%s", printErr)
+	}
+
+	err = compileBuildFailureErrors(err, allowToolchainRebuilds, buildState)
+
+	return
```

Comment:
Thoughts on this?

```suggestion
	return builtGraph, compileBuildFailureErrors(err, allowToolchainRebuilds, buildState)
```

Reply:
I like it but to do it "right", I think I should update all `return` statements at least in this file and I wouldn't want to include a refactor with this change.